src/main/resources/xml/schema/2020-1/nordic-html5.rng

```diff
@@ -2088,55 +2087,66 @@
             </attribute>
         </optional>
     </define>
-    
+
     <define name="poem">
-        <!-- Use: poem is a complete poem or fragment thereof. -->
         <a:documentation>  Use: poem is a complete poem or fragment thereof.  </a:documentation>
-        <element name="section">
-            <!-- HTML content model: (a | abbr | address | article | aside | audio | b | bdi | bdo | blockquote | br | button | canvas | cite | code | data | datalist | del | details | dfn | dialog | div | dl | em | embed | fieldset | figure | footer | form | h1 | h2 | h3 | h4 | h5 | h6 | header | hr | i | iframe | img | input | ins | kbd | keygen | label | link | main | map | mark | math | menu | meta | meter | nav | noscript | object | ol | output | p | pre | progress | q | ruby | s | samp | script | section | select | small | span | strong | style | sub | sup | svg | table | template | textarea | time | u | ul | var | video | wbr | (text))* -->
-            <!-- Strict content model: (h1 | span | p | section | img | figure)* -->
-            <ref name="attlist.poem"/> <!-- @epub:type, @class, @id, @title, @xml:space, @xml:lang, @lang, @dir -->
-            <zeroOrMore>
+        <element name="div">
+            <attribute name="class">
+                <value>verse</value>
+            </attribute>
+            <optional>
                 <choice>
                     <ref name="title"/> <!-- h1, h2, h3, h4, h5, h6 -->
-                    <ref name="author.block"/> <!-- p -->
-                    <ref name="hd"/> <!-- h1, h2, h3, h4, h5, h6 -->
-                    <ref name="dateline"/> <!-- span -->
-                    <ref name="epigraph"/> <!-- blockquote, p -->
-                    <ref name="byline"/> <!-- span -->
-                    <ref name="linegroup"/> <!-- div, section -->
-                    <ref name="line"/> <!-- p -->
-                    <ref name="pagebreak.block"/> <!-- div -->
-                    <ref name="img"/> <!-- img -->
-                    <ref name="figure.imggroup"/> <!-- figure -->
-                    <ref name="sidebar"/> <!-- aside, figure -->
+                    <ref name="bridgehead"/>
                 </choice>
-            </zeroOrMore>
+            </optional>
+            <ref name="poem.linegroup"/>
+            <optional>
+                <element name="p">
```

Comment:
Poems are required to have bridgheads (poem-title) before any linegroup, and verse-author after all the linegroups. This is probably how a poem is normally structured, but it is not described in the guidelines and I suppose there could be cases where the order could be different. For instance if you want to list the author right after the bridgehead/verse-title/header, if a verse (in the sense "linegroup") has a title (i.e. bridgehead) of its own, or is written by a different author.

@martinpub @AndersEkl : what do you think? Should there be a required order here?

If we allow zero or more bridgeheads and zero or more authors, we could do something like:

```xml
<element name="div">
    <attribute name="class">
        <value>verse</value>
    </attribute>
    <zeroOrMore>
        <choice>
            <ref name="bridgehead"/>
            <!-- verse-author here -->
        </choice>
    </zeroOrMore>
    <ref name="poem.linegroup"/>
    <zeroOrMore>
        <choice>
            <ref name="poem.linegroup"/>
            <ref name="bridgehead"/>
            <!-- verse-author here -->
        </choice>
    </zeroOrMore>
</element>
```

Reply:
I think it is reasonable to require that a bridgehead, if there is one, is the first child of the `<div class="verse">` element. I am not sure it is necessary to have any rules concerning placement of the author? But it should not be inside a linegroup.

Reply:
Ok. Should we wait for feedback from @martinpub if he has an opinion about the placement of the author?

Reply:
Maybe we shouldn't be too strict on these rules? What if you have a couple of poems in a row, with separate bridgeheads and separate authors? Should we then require a separate `<div>` for each poem in order to enforce rules about placement of bridgehead and author? Or should we allow a bit of flexibilty? It may be enough to just check that neither bridgehead or author are placed inside a linegroup? But maybe @martinpub should say what he thinks as well?

Reply:
Could each verse potentially have a bridghead? Either if they actually have a title, or just to say "Verse 1", "Verse 2" etc. (if that's how it's done in the original or requested in editing instructions).

Reply:
Yeah, I don't see why not? And what would be the real gain from having very strict rules about where to place a bridgehead or an author name inside a verse? Is it necessary? We don't have any rules about placement of bridgeheads elsewhere, right?

Reply:
I think the only rule about bridgeheads in other places is that they can only occur as a direct child of `<section>` (same as for normal headings), but I'm not sure.